navigation/AppNavigation.js

```diff
@@ -1,25 +1,29 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
-import { createStackNavigator, createAppContainer } from 'react-navigation';
+import { createStackNavigator, createAppContainer, createBottomTabNavigator } from 'react-navigation';
 
-import { white } from '../scss/colors';
+import { white, lightBlue, gray } from '../scss/colors';
+import IconComponent from '../app/components/baseComponents/IconComponent';
 import LibraryContainer from '../app/screens/Library/index';
 import BookDetailContainer from '../app/screens/BookDetail/index';
+import WishlistContainer from '../app/screens/WishList';
 
 import ImageHeader from './components/ImageHeader';
 import BackButtonHeader from './components/BackButtonHeader';
+import { ROUTES } from './constants';
 
-const AppNavigator = createStackNavigator(
+const LibraryStack = createStackNavigator(
   {
     library: {
       screen: LibraryContainer,
       navigationOptions: {
-        title: 'Library'
+        title: 'LIBRARY'
```

Comment:
you can extract this `title` property in a common `defaultTabNavigationOpcion` and use `navigation.state.routeName` and `headerTitle` and avoid explicitly stating the tile like this